dev-docs/modules/userid-submodules/ceeIdSystem.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: userid
+title: CEEIdSystem UserID Module
+description: CEEID User ID sub-module
+useridmodule: ceeIdSystem
+---
+
+## Prebid Configuration
+
+First, make sure to add ceeIdSystem to your Prebid.js package with:
```

Comment:
Normally there's a need for a publisher to register. Do they need to contact you first? How?

Reply:
This PR was only requested to do some changes in already existing documentation (https://docs.prebid.org/dev-docs/modules/userid-submodules/ceeIdSystem.html). Not sure why it shows like its a new file. They don't need to register or contact us first we contact with them.

Reply:
Previous PR: https://github.com/prebid/prebid.github.io/pull/5449

Reply:
yeah, odd. Don't see why it thinks this is a new file.